### PR TITLE
Describe what a session secret is in the Auth0 auth provider

### PR DESCRIPTION
**Pull request by waldirmontoya25**
## Hey, I just made a Pull Request!

Describe what a session secret is in the Auth0 auth provider

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/auth/auth0/provider.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The Auth0 provider is a structure with three configuration keys:
 - `clientSecret`: The Application client secret, found on the Auth0 Application
   page
 - `domain`: The Application domain, found on the Auth0 Application page
+- `sessionsecret`: The session secret is a key used for signing and/or encrypting cookies set by the application to maintain session state. In this case, 'your session secret' should be replaced with a long, complex, and unique string that only your application knows.
```
Comment (jamieklassen) [inline docs/auth/auth0/provider.md:50]:
```suggestion
- `session.secret`: The session secret is a key used for signing and/or encrypting cookies set by the application to maintain session state. In this case, 'your session secret' should be replaced with a long, complex, and unique string that only your application knows.
```

It kinda feels like this should be documented at a higher level somewhere? This piece of configuration is independent of any particular auth provider...

**Review by freben — COMMENTED** [review]
Thanks! One tweak suggested.

**Comment by freben** [comment]
Sorry, that build error was introduced just recently and is fixed by https://github.com/backstage/backstage/pull/18938

**Review by freben — APPROVED** [review]
Cool!

**Pull request merged**